srcpkgs/neomutt/template

```diff
@@ -19,11 +19,12 @@ short_desc="Mutt with misc fixes and feature patches"
 maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://neomutt.org/"
-_test_files_hash=34f79ee9d91863a97f6e18700aea51157b8610cf
+_test_files_hash=00efc8388110208e77e6ed9d8294dfc333753d54
```

Comment:
I mean I guess you could do this... the reason I'd ignored it is the latest commit is a documentation only update. Not actually functionally required.

Reply:
But I'll accept this.